content/en/ingress.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Ingress
+status: Feedback Appreciated
+category: technology
+tags: ["fundamental", "", ""]
+---
+
+## What it is
+
+An Ingress is an abstraction that helps to manage the traffic from the outside into a [Kubernetes](/kubernetes/) cluster. 
+It consists of two elements, the ingress resource, and the ingress controller. 
+The ingress resource is just a Kubernetes manifest file that lives along with other manifest files and allows for the configuration of traffic routing. 
```

Comment:
The term "Kubernetes manifest file" here might be a bit technical for our target audience (assuming knowledge of Kubernetes)

Reply:
I feel in this case either we can use the term configuration file or just explain Kubernetes manifest file in next line. What you would suggest?

Reply:
I'd rather generalise the ingress concept than have to explain kubernetes

Reply:
Okay got it, Thanks!